fastchat/conversation.py

```diff
@@ -224,6 +225,15 @@ def get_prompt(self) -> str:
                     ret += role + ":"
 
             return ret
+        elif self.sep_style == SeparatorStyle.DEEPSEEK_CHAT:
+            seps = [self.sep, self.sep2]
+            ret = system_prompt
+            for i, (role, message) in enumerate(self.messages):
+                if message:
+                    ret += role + ": " + message + seps[i % 2]
+                else:
+                    ret += role + ":"
+            return ret
```

Comment:
just to confirm, there's no existing template suitable for deepseek?

Reply:
The closest I could find is `SeparatorStyle.ADD_COLON_TWO`, the only issue is that it has an extra `system_prompt + seps[0]` instead of just `system_prompt`, which leads to an extra `\n\n` if we use that conversation style. 

The difficulty comes from:
1. We need to prepend a `<|begin_of_sentence|>` token before the user response which I think the easiest location to put that is the system_message.
2. There is an extra `<|end_of_sentence|>` after the Assistant's message, which requires us to have two steps

TLDR: I couldn't find another suitable template

Reply:
got it, thanks! moving forwrad, we should consider migrating to HF's chat template so we can leave this responsibility to model developers.
https://huggingface.co/deepseek-ai/deepseek-llm-67b-chat/blob/79648bef7658bb824e4630740f6e1484c1b0620b/tokenizer_config.json#L34